jira_automation/README.md

```diff
@@ -0,0 +1,25 @@
+There are three applications in here to facilitate our JIRA/GitHub interaction
+
+1. io.dockstore.jira.ResolutionChecker - used to help find issues open in GitHub that are closed in JIRA. This was to diagnose an issue where 
```

Comment:
May make more sense to order most useful to least useful. 
i.e. not have application we're likely to not use first